Sources/Model/HyperwalletBankAccount.swift

```diff
@@ -64,30 +111,39 @@ public final class HyperwalletBankAccount: HyperwalletTransferMethod {
         /// - Parameters:
         ///   - transferMethodCountry: The bank account country.
         ///   - transferMethodCurrency: The bank account currency.
-        ///   - transferMethodProfileType: The method profile type
-        public init(transferMethodCountry: String, transferMethodCurrency: String, transferMethodProfileType: String) {
+        ///   - transferMethodProfileType: The bank account holder's profile type 'ProfileType'
+        public init(transferMethodCountry: String, transferMethodCurrency: String, transferMethodProfileType: ProfileType) {
```

Comment:
keep it as String instead of ProfileType

Reply:
Reverted